source/unified-test-format/schema-latest.json

```diff
@@ -369,6 +378,28 @@
         "connection"
       ]
     },
+    "spanComponent": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "tags"
```

Comment:
If there are some cases where you do not intend to assert tags (thinking of the transaction test), should this field _not_ be required?

Reply:
Updated, this field is now named `attributes` and should have at least `db.system: mongodb` reported, so keeping it required.